user-guide/Advanced_Functionality/DataMiner_Agents/Upgrading_a_DMA/Upgrading_Downgrading_Webapps.md

```diff
@@ -22,3 +22,19 @@ After the upgrade process, your dashboards and low-code apps may be migrated to
 
 > [!NOTE]
 > The *Backups* folder retains only the five most recent backup files.
+
+## Downgrading the DataMiner web apps
+
+Since dashboards and low-code apps can automatically get migrated after a software upgrade, these will then no longer work in combination with older versions of the software.
+
+When downgrading the software, these steps need to be executed manually:
```

Comment:
Is this something you should do after the downgrade is done?

Reply:
Indeed, otherwise when doing this before the downgrade there's a chance that the dashboard/lca got migrated again.